### DISCUSSION TITLE
为啥open-api请求接口拿不到任何数据

### DISCUSSION THREAD

**Discussion by storehubdeploy** [apolloconfig/apollo#4250] [Q&A]
请求接口 http://{portal_address}/openapi/v1/envs/{env}/apps/{appId}/clusters/{clusterName}这个显示404, 加上namespaces可以拿到namespace的数据。 这是为啥？

**Comment by nobodyiam** [comment]
You may check the log of apollo-portal and apollo-adminservice for more information.

**Reply by storehubdeploy** [reply]
can not find any message in logs
